README.md

```diff
@@ -1,160 +1,178 @@
+- Your OP-Stack chain is running and has at least one finalized block. This is important because the Babylon fast finality gadget starts processing blocks from the (non-zero) finalized height.
 
+For more details about how to setup an OP-Stack chain with BTC staking support, please refer to the [OP chain deployment](https://github.com/Snapchain/op-chain-deployment/blob/main/README.md) repo.
 
-Firstly, please get some test tokens from the Euphrates faucet
+### 1. Get some test BBN tokens from the Euphrates faucet
 
 ```bash
 curl https://faucet-euphrates.devnet.babylonlabs.io/claim \
 -H "Content-Type: multipart/form-data" \
 -d '{ "address": "<YOUR_BABYLON_ADDRESS>"}'
 ```
 
-### 1. Setup environment variables
+### 2. Setup environment variables
 
 Copy the `.env.babylon-integration.example` file to `.env.babylon-integration`
 
 ```bash
 cp .env.babylon-integration.example .env.babylon-integration
 ```
 
-**Configure for Bitcoin**
-
-Based on the previous step `Setup Bitcoin node`, set the following variables with the values:
-- `BITCOIN_RPC_PASS`
-- `BTC_WALLET_PASS`
-    
-and replace the IP with your Bitcoin node IP:
-- `BITCOIN_RPC_HOST`
-- `ZMQ_RAWBLOCK_URL`
-- `ZMQ_RAWTX_URL`
-
-**Configure for Babylon**
+The key env vars to set are the server IP addresses where your Bitcoin Signet node and Babylon finality system are deployed. Note that the ports are preconfigured in the Docker Compose files used for this deployment.
 
-set your Babylon key's mnemonic, the address should have some BBN tokens, it will be used in the following steps
+- `BITCOIN_RPC_HOST`: the Bitcoin Signet node's IP address.
+- `ZMQ_RAWBLOCK_URL`: the Bitcoin Signet node's IP address.
+- `ZMQ_RAWTX_URL`: the Bitcoin Signet node's IP address.
+- `CONSUMER_EOTS_MANAGER_ADDRESS`: the Babylon finality system's IP address.
+- `FINALITY_GADGET_RPC`: the Babylon finality system's IP address.
+- `NEXT_PUBLIC_FINALITY_GADGET_API_URL`: the Babylon finality system's IP address.
 
-- `BABYLON_PREFUNDED_KEY_MNEMONIC`
+Besides these, you will need to set the following variables:
 
-set the following variables to register your OP Stack chain to Babylon:
+- `BABYLON_PREFUNDED_KEY_MNEMONIC`: the mnemonic for the wallet you used to claim BBN tokens in the previous step.
+- `CONSUMER_ID`: this is the identifier for your OP-Stack chain registration on Babylon, you can set it to anything you want (the convention we use is `<chain_type>-<chain_name>-<chain_id>-<version>`, e.g. `op-stack-tohma-706114-0001`).
+- `CONSUMER_CHAIN_NAME`: this is a human-readable name for your chain.
+- `OP_FP_MONIKER`: this is a human-readable name for your OP-Stack chain's finality provider.
+- `L2_RPC_URL`: this is your OP-Stack chain's RPC URL.
 
-- `CONSUMER_ID`
-- `CONSUMER_CHAIN_NAME`
+### 3. Set Babylon keys
 
-set your OP Stack chain's finality provider moniker
+This step imports the pre-funded Babylon key, which will be used to deploy the different components of the Babylon finality system:
```

Comment:
@parketh it's a bit weird to say the pre-funded key is used to deploy BTC staker. it's used by the BTC staker to do btc delegation

same for EOTS / FP, the key is not to deploy those components